### PR TITLE
Various improvements to install.sh

### PR DESCRIPTION
**Pull request by Thiht**
- Fix [Shellcheck](https://www.shellcheck.net/) warnings
- Ask for permission to writ /usr/local/bin
- Fix some issues from jpillora/installer
- Retrieve latest version

### CODE REVIEW

**Comment by xizhao** [comment]
Thank you for this contribution! @ilikebits can you review?

File: install.sh [hunk 1]
```diff
@@ -11,52 +11,61 @@
...
-  cleanup
   msg=$1
   echo "============"
   echo "Error: $msg" 1>&2
   exit 1
 }
 
+# This function will ask for root privileges before executing a command
+# The goal is to allow the used to run this script as a normal user and
+# to ash authorizations as needed
```
Comment (waldyrious) [inline install.sh]: Some minor corrections:
- "allow the used to run" --> "allow the user to run"
- "to ash authorizations as needed" --> "to be asked for authorizations as needed"

Comment (Thiht) [reply install.sh]: fixed in b67280f :)

**Comment by ilikebits** [comment]
Merging with admin override:
- CircleCI and FOSSA are temporarily broken for PRs due to the `$FOSSA_API_KEY` usage: tracking at #115.

**Pull request merged**